client/src/components/GiveForm.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import { useForm } from "react-hook-form";
+import { Redirect } from "react-router-dom";
+import { ErrorMessage } from "@hookform/error-message";
+
+import WarmFuzzyClient from "../apiClient/WarmFuzzyClient";
+import Authentication from "../providers/Authentication";
+
+const GiveForm = () => {
+  const { user, createNewFuzzy } = Authentication.useAuthentication();
+  const [shouldRedirect, setShouldRedirect] = useState(false);
+  const { handleSubmit, register, errors, reset } = useForm();
+
+  const onSubmit = async (data) => {
+    const client = new WarmFuzzyClient();
+    const warmFuzzyData = await client.postFuzzy({ ...data, elevatorId: user.userId });
+    if (warmFuzzyData) {
+      setShouldRedirect(true);
+    }
+  };
+
+  if (shouldRedirect) {
+    return <Redirect push to="/warm-fuzzies/index" />;
```

Comment:
the react route here should likely be `/warm-fuzzies` to adhere to RESTful conventions